npdarrington/ideabox-boilerplate
# Nd finalizeproject

PR Templates:

What’s this PR do?  
- Finalize README.md
- Completed styling for comments
 
Where should the reviewer start? 
- Open index.html
 
How should this be manually tested?  
- Add comments after clicking on the plus icon.
 
Any background context you want to provide?  
N/A
 
What are the relevant tickets?  
N/A
 
Screenshots (if appropriate)  
Please see README.
